nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-api/src/main/java/org/nuxeo/ecm/platform/comment/api/CommentManager.java

```diff
@@ -50,28 +79,57 @@
      * @param comment the comment content
      * @param author the comment author
      * @return the comment document model.
+     * @deprecated since 10.3, use {@link #createComment(CoreSession, Comment)} instead.
      */
+    @Deprecated
     DocumentModel createComment(DocumentModel docModel, String comment, String author);
 
+    /**
+     * Creates a comment document model, filling its properties with given info and linking it to given document.
+     *
+     * @param docModel the document to comment
+     * @param comment the comment document model
+     * @return the created comment document model.
+     */
     DocumentModel createComment(DocumentModel docModel, DocumentModel comment);
 
+    /**
+     * Creates a comment document model, filling its properties with given info and linking it to given document.
+     *
+     * @param docModel the document to comment
+     * @param parent the comment parent document model
+     * @param child the comment child document model
+     * @return the created comment document model.
+     * @deprecated since 10.3, use {@link #createComment(CoreSession, Comment)} instead.
+     */
+    @Deprecated
     DocumentModel createComment(DocumentModel docModel, DocumentModel parent, DocumentModel child);
 
+    /**
+     * Deletes a comment.
+     *
+     * @param docModel the comment document model
+     * @param comment the comment
+     * @deprecated since 10.3, use {@link #deleteComment(CoreSession, String)} instead.
+     */
+    @Deprecated
     void deleteComment(DocumentModel docModel, DocumentModel comment);
 
     /**
      * Gets documents in relation with a particular comment.
      *
      * @param comment the comment
      * @return the list of documents
+     * @deprecated since 10.3
      */
+    @Deprecated
     List<DocumentModel> getDocumentsForComment(DocumentModel comment);
 
     /**
-     * Gets thread in relation with a given comment (post or comment)
+     * Gets thread in relation with a given comment (post or comment).
      *
-     * @param comment
-     * @return
+     * @param comment the comment
+     * @return the thread
      * @since 5.5
      */
     DocumentModel getThreadForComment(DocumentModel comment);
```

Comment:
Could be deprecated no ?

Reply:
It is actually used by the `CommentNotificationListener`